startbootstrap-sb-admin-2-gh-pages/server.js

```diff
@@ -42,14 +44,24 @@ app.get("/register.html", (req, res) => {
 });
 
 app.post("/register", async (req, res) => {
-  console.log(req.body)
   var isUserRegistered = await TryRegisterUser(req.body, res);
   if (!isUserRegistered) {
     return res.send({valid: false})
   }
   return res.send({valid: true})
 });
 
+app.post("/check-email", async (req, res) => {
+  const connection = GetUserDatabase(mysql);
+  var does = await doesUserExist(connection, req.body.email);
```

Comment:
maybe give this a better name? I feel like 'does' is a bit vague even if it is only used for this.